templates/taxi/driver_list.html

```diff
@@ -4,11 +4,18 @@
   <h1>Driver list</h1>
```

Comment:
Checklist item: Code style. Please use 2 whitespaces for indentation in your `.html` files.